functions.php

```diff
@@ -44,6 +44,18 @@ function fm_get_byline_posts( $byline_id, $type = 'author', $args = array() ) {
 }
 
 /**
+ * Get the WP Posts Query for a byline user.
+ * @param int $byline_id. Defaults to current displayed byline id.
+ * @param string $type. Defaults to current display byline type.
+ * @param array $args.
+ * @return mixed. WP Query object or false.
+ */
+function fm_get_byline_posts_query( $byline_id = null, $type = 'author', $args = array() ) {
```

Comment:
Yeah, the idea is to use this on a byline page and it can correctly guess the byline type and byline id by query vars and post ids.....